### PR TITLE
Some more tidying up of CV code

### PR DESCRIPTION
**Pull request by chrisforbes**

### CODE REVIEW

File: layers/core_validation.cpp [hunk 1]
```diff
@@ -9731,10 +9731,10 @@ VKAPI_ATTR void VKAPI_CALL CmdExecuteCommands(VkCommandBuffer commandBuffer, uin
                 pCB->queryUpdates.push_back(function);
             }
         }
-        skip |= validatePrimaryCommandBuffer(dev_data, pCB, "vkCmdExecuteComands", VALIDATION_ERROR_00163);
-        skip |= ValidateCmdQueueFlags(dev_data, pCB, "vkCmdExecuteComands()",
+        skip |= validatePrimaryCommandBuffer(dev_data, pCB, "vkCmdExecuteCommands", VALIDATION_ERROR_00163);
```
Comment (tobine) [inline layers/core_validation.cpp]: Should also add "()" to end of function name here

File: layers/core_validation.cpp [hunk 2]
```diff
@@ -7148,7 +7074,10 @@ VKAPI_ATTR void VKAPI_CALL CmdBindDescriptorSets(VkCommandBuffer commandBuffer,
                                     (uint64_t)pDescriptorSets[set_idx]);
                 }
                 // Verify that set being bound is compatible with overlapping setLayout of pipelineLayout
-                if (!verify_set_layout_compatibility(dev_data, descriptor_set, pipeline_layout, set_idx + firstSet, error_string)) {
+                if (!verify_set_layout_compatibility(descriptor_set,
```
Comment (mark-lunarg) [inline layers/core_validation.cpp]: These guys look like they might need to run through clang-format

**Comment by mark-lunarg** [comment]
+1, the first commit message needs to be reformatted.

File: layers/core_validation.cpp (hunk 2 above)
Comment (chrisforbes) [inline layers/core_validation.cpp]: Fixed, thanks.

**Pull request merged**